operations/network/management_network/backup_custom_configurations.md

```diff
@@ -57,93 +57,93 @@ interface 1/1/36
     exit
 ```
 
-```
+```console
 sw-spine-002(config)# show run | include interface-group
 system interface-group 3 speed 10g
 ```
 
-```
+```console
 sw-spine-001# show run | include "ip route"
 ip route 0.0.0.0/0 10.101.15.141
 ```
 
-```
+```console
 sw-spine-002# show run | include "ip route"
 ip route 0.0.0.0/0 10.101.15.189
 ```
 
-### Mellanox
+### Mellanox site connection
 
-```
+```console
 sw-spine-001 [mlag-domain: master] # show run int ethernet 1/16
 interface ethernet 1/16 speed 10G force
 interface ethernet 1/16 mtu 1500 force
 interface ethernet 1/16 no switchport force
 interface ethernet 1/16 ip address 10.102.255.10/30 primary
 ```
 
-```
+```console
 sw-spine-002 [mlag-domain: master] # show run int ethernet 1/16
 interface ethernet 1/16 speed 10G force
 interface ethernet 1/16 mtu 1500 force
 interface ethernet 1/16 no switchport force
 interface ethernet 1/16 ip address 10.102.255.86/30 primary
 ```
 
-```
+```console
 sw-spine-001 [mlag-domain: master] # show run | include "ip route"
    ip route 0.0.0.0/0 10.102.3.3 5
    ip route 0.0.0.0/0 10.102.255.9
 ```
 
-```
+```console
 sw-spine-002 [mlag-domain: master] # show run | include "ip route"
    ip route 0.0.0.0/0 10.102.3.2 5
    ip route 0.0.0.0/0 10.102.255.85
 ```
 
 ## Backup users/passwords
```

Comment:
This still needs to be resolved: https://github.com/Cray-HPE/docs-csm/pull/1688#discussion_r881999420